### PR TITLE
Fix adj_to_csr_kernel 

### PR DESCRIPTION
**Pull request by ahendriksen**
This PR adds launch bounds to adj_to_csr kernel, making it forward compatible with future architectures.

### CODE REVIEW

File: cpp/include/raft/sparse/convert/detail/adj_to_csr.cuh [hunk 1]
```diff
@@ -29,6 +29,9 @@ namespace sparse {
 namespace convert {
 namespace detail {
 
+// Threads per block in adj_to_csr_kernel.
+const int adj_to_csr_tpb = 512;
```
Comment (wphicks) [inline cpp/include/raft/sparse/convert/detail/adj_to_csr.cuh:33]:
```suggestion
static const constexpr int adj_to_csr_tpb = 512;
```

Comment (ahendriksen) [reply cpp/include/raft/sparse/convert/detail/adj_to_csr.cuh]: Thanks! Changed.

**Review by wphicks — APPROVED** [review]

**Review by tfeher — APPROVED** [review]
Thanks @ahendriksen for the PR, it looks good to me!

**Comment by wphicks** [comment]
@gpucibot merge

**Pull request merged**